0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/GenerateCommand.scala

```diff
@@ -92,6 +93,20 @@ case class GenerateCommand(
     // Parse Self-describing Schemas
     val (schemaErrors, schemas) = splitValidations(files.map(_.extractSelfDescribingSchema))
 
+    // Check schema versions
+    val versions: List[String] = schemas.map(schema => schema.self.version.asString).sorted
+    val schemaVerWarning: List[String] =
+      if (input.isFile && !versions.contains("1-0-0"))
+        List(s"Warning: File [${input.getAbsolutePath}] contains a schema whose version is NOT 1-0-0")
+      else List.empty
+    val schemaVerError: List[String] =
+      if(input.isDirectory && versions.head != "1-0-0" && !force)
+        List(s"Error: Directory [${input.getAbsolutePath}] does NOT contain a schema whose version is 1-0-0. You can use --force to override.")
+      else if(input.isDirectory && versions.head == "1-0-0" && existMissingSchemaVersion(versions) && !force){
```

Comment:
Again, directory `com.acme` with:

* `com.acme/event/jsonschema/1-0-0`
* `com.acme/context/jsonschema/1-0-1`
* `com.acme/context/jsonschema/1-0-2`

has no gaps (`List(1-0-0, 1-0-1, 1-0-2)`) according to this function, but obviously incorrect overall.

Think of this function working with `schemas` directory from https://github.com/snowplow/iglu-central, not `jsonschema` from https://github.com/snowplow/iglu-central/tree/master/schemas/com.mailchimp/cleaned_email